Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -407,6 +391,16 @@ private void InitializeServiceLocator()
                 }
             }
 
+            // If the Camera system has been selected for initialization in the Active profile, enable it in the project
+            if (ActiveProfile.IsCameraSystemEnabled)
+            {
+                object[] args = { this, ActiveProfile.CameraProfile };
+                if (!RegisterService<IMixedRealityCameraSystem>(ActiveProfile.CameraSystemType, args: args) || CameraSystem == null)
```

Comment:
What the heck is going on here?
Is this a product of the new registrar system?

Reply:
This is how other core services are registered. @davidkline-ms could probably tell you more.